lldb/source/Plugins/ExpressionParser/Swift/SwiftExpressionParser.cpp

```diff
@@ -622,8 +630,8 @@ static void AddRequiredAliases(Block *block, lldb::StackFrameSP &stack_frame_sp,
     LLDB_LOG(GetLog(LLDBLog::Types | LLDBLog::Expressions),
              "Couldn't get SwiftASTContext type for self type {0}.",
              imported_self_type.GetDisplayTypeName());
-
-    return;
+    return Status(
+        "Could not add required aliases: could not get Swift type for self.");
```

Comment:
```suggestion
        "Unable to add the aliases the expression needs because the Swift expression parser couldn't get the Swift type for `self`."
```

If the word "Unable" isn't a valid option, then I suggest the contraction "Couldn't".